File: Python Excel Examples/WorksheetsApiDemo/searchText.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "SearchText.xlsx"
folder = "ExcelDocument";
storage = ""
sheetName = "Sheet1"
text = "America"
matchCase = "true"  # Default value is false
result = api.search_text(name, sheetName, text, match_case=matchCase, folder=folder, storage=storage)

File: PHP Excel Examples/WorksheetsApiDemo/searchText.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "SearchText.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet4";
$text = "America";
#default value is false
$matchCase = false;
$result = $apiInstance->searchText($name, $sheetName, $text, $matchCase,$folder, $storage);